{"uuid":"b0adadc2-791c-499b-a5d3-7424dd6391a9","name":"Scenario Outline: The user is presented with error message for invalid code in tryEditor for Max Consecutive Ones","children":["8fbf2a20-b447-496c-8109-88f81978624d"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696879569745,"stop":1696879569745},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696879569892,"stop":1696879569893},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696879570009,"stop":1696879570009},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696879572050,"stop":1696879572050}],"start":1696879569182,"stop":1696879572051}